.github/workflows/ar-chromatic.yml

```
@@ -0,0 +1,52 @@
+name: AR Chromatic 👓
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  chromatic:
+    name: Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout - On Pull Request
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'pull_request'}}
+        with:
+          fetch-depth: 0
+          # By default the pull_request event will run on a ephermeral merge commit which simulates a merge between the pull request
+          # and the target branch. This can cause issues with Chromatic https://www.chromatic.com/docs/turbosnap#github-pullrequest-triggers
+          # Hopefully by checking out the HEAD commit of a PR instead of the merge commit we can avoid some of those issues.
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Checkout - On Push Event
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push'}}
+        with:
+          fetch-depth: 0
+
+      - uses: guardian/actions-setup-node@main
```

I think this action is deprecated now (https://github.com/guardian/actions-setup-node/issues/9)! We can use `actions/setup-node@v3`:

```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
          cache: 'yarn'
```

---

Ah interesting - seems like a sensible time to make the change.